src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/MockGoogleDataprocDAO.scala

```diff
@@ -12,12 +12,18 @@ class MockGoogleDataprocDAO extends DataprocDAO {
 
   private val clusters: mutable.Map[String, Cluster] = new TrieMap()
 
+  private val googleID = UUID.randomUUID().toString
+
   override def createCluster(googleProject: String, clusterName: String, clusterRequest: ClusterRequest)(implicit executionContext: ExecutionContext): Future[ClusterResponse] = {
-    val clusterResponse = ClusterResponse(clusterName, googleProject, UUID.randomUUID().toString, "status", "desc", "op-name")
+    val clusterResponse = ClusterResponse(clusterName, googleProject, googleID, "status", "desc", "op-name")
```

Comment:
if a test calls `createCluster` twice both clusters will have the same google uuid? this seems wrong